front-end/src/components/player-page/stats.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import {
+    Box,
+    FormControl,
+    Grid,
+    InputLabel,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+    Theme,
+    Typography
+} from "@mui/material";
+import RisenBox1 from "../risen-box/risen-box-1";
+import PerformanceOverview from "./stats/performance-overview";
+import {GameRoles} from "../../../../Common/Interface/General/gameEnums";
+import RoleSelector from "../selectors/role-selector";
+import RisenSeasonSelector from "../selectors/risen-season-selector";
+import SeasonModel from "../../../../Common/models/season.model";
+import FilterBar from "./stats/filter-bar";
+
+interface PlayerPageStatsProps {
+    seasonConfig?: {
```

Comment:
Probably should be mandatory. idk how this works without the config. Cuz we at least need the setSeasonId functiono

Reply:
Yea its WIP